### ISSUE TITLE
Compile the pod with Swift 4.1

### ISSUE THREAD

**Issue by frootloops** [onfido/onfido-ios-sdk#24]

**Comment by Gorbyy** [comment]
Hi @frootloops 
At the moment, we only support Swift 3.2.2 and Swift 4.0.2.
Best

**Comment by dim-bon** [comment]
Any timeframes for when an update is going to be available?

**Comment by anuragajwani** [comment]
We are currently looking into it and aiming to have the SDK compiled with Swift 4.1 compiler by the end of this week.

Anurag

**Comment by anuragajwani** [comment]
@frootloops @dim-bon we have released SDK version 6.0.0 compatible with Swift 3.3 and Swift 4.1

Anurag

**Comment by olliem36** [comment]
@anuragajwani Thanks!

**Comment by yoginair** [comment]
> @frootloops @dim-bon we have released SDK version 6.0.0 compatible with Swift 3.3 and Swift 4.1
> 
> Anurag

Where in release tab i can see only two releases ?

**Comment by anuragajwani** [comment]
@yoginair we started supporting Github releases on 10.1.0. 
Onfido SDK 6.0.0 has some dependencies. You can find the spec [here](https://github.com/CocoaPods/Specs/blob/master/Specs/c/9/3/Onfido/6.0.0/Onfido.podspec.json).

Dependencies:
```
"dependencies": {
    "Alamofire": [
      "< 4.5"
    ],
    "MBProgressHUD": [
      "~> 1.0.0"
    ],
    "Swinject": [
      "< 2.3"
    ],
    "SwiftyJSON": [
      "~> 3.1"
    ]
  }
```

- [debug framework](https://s3-eu-west-1.amazonaws.com/onfido-sdks/ios/Onfido-Debug-v6.0.0.zip)
- [release framework](https://s3-eu-west-1.amazonaws.com/onfido-sdks/ios/Onfido-Release-v6.0.0.zip)
